src/main/resources/config/liquibase/changelog/20200213171522_changelog.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="jonathantumboimbela" id="20200213171522">
+        <addColumn tableName="course">
+            <column name="max_complaint_time_weeks" type="integer"/>
+        </addColumn>
+        <addColumn tableName="course">
+            <column name="qna_enabled" type="Boolean" defaultValueBoolean="true">
```

Comment:
please delete the old changelog and make sure the new one is atomic, so only adding the new column, no deletions